### PR TITLE
Task/13 add cta for create po

### PR DESCRIPTION
**Pull request by mllrdev-jerry-bayoneta**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <template>
   <ion-app>
     <ion-router-outlet />
+    <h1>Hello world</h1>
```
Comment (mllrdev-carlo-medel) [inline src/App.vue]: Do you need this?

File: src/core/components/Header.vue [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<template>
```
Comment (mllrdev-carlo-medel) [inline src/core/components/Header.vue]: Transfer to shared folder

File: src/core/interfaces/order-detail.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import IItem from "@/core/interfaces/item";
+import IPurchaseOrder from "@/core/interfaces/purchase-order"
+
+interface IOrderDetail {
+  item: IItem[],
+  supplier: string,
+  purchaseDate: Date,
+  grandTotal: number,
+  description: IPurchaseOrder['description']
```
Comment (mllrdev-carlo-medel) [inline src/core/interfaces/order-detail.ts]:
```suggestion
  description: string
```

File: src/pages/Create.vue [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+            v-model="item.subTotal"
+            name="subTotal"
+            placeholder="Total Price"
+            disabled
+          ></ion-input>
+        </ion-row>
+        <ion-button button type="submit">ADD</ion-button>
+      </form>
+      <ion-text color="light">
+        <h3>Grand Total {{orderDetail.grandTotal}}</h3>
```
Comment (mllrdev-carlo-medel) [inline src/pages/Create.vue]:
```suggestion
        <h3>Grand Total {{ orderDetail.grandTotal }}</h3>
```

File: src/pages/Create.vue [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+    addItemDetail() {
+      this.item.subTotal = this.item.unitPrice * this.item.quantity;
+      this.itemDetails.push(this.item);
+      
+      this.item = {
+        name: "",
+        quantity: 0,
+        unitPrice: 0,
+        subTotal: 0,
+      };
```
Comment (mllrdev-carlo-medel) [inline src/pages/Create.vue]: Initialize to null if want to have a "nothing"

File: src/shared/components/OrderItem.vue [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'OrderItem',
+  setup() {
+
+    
+  }
+
+
```
Comment (mllrdev-carlo-medel) [inline src/shared/components/OrderItem.vue]:
```suggestion 
  }
```

File: src/types/purchaseOrder.ts [hunk 7]
```diff
@@ -0,0 +1,10 @@
+interface PurchaseOrder {
+    purchaseOrderNumber: string,
+    description: string,
+    date: Date,
+    status: 'pending' | 'approved',
+    action: 'reviewed'| 'for review'
```
Comment (mllrdev-carlo-medel) [inline src/types/purchaseOrder.ts]: Convert to enum

Comment (mllrdev-carlo-medel) [inline src/types/purchaseOrder.ts]: Convert to enum

File: src/core/components/Header.vue (hunk 2 above)
Comment (mllrdev-jerry-bayoneta) [reply src/core/components/Header.vue]: moved to shared folder

File: src/core/interfaces/order-detail.ts (hunk 3 above)
Comment (mllrdev-jerry-bayoneta) [reply src/core/interfaces/order-detail.ts]: changed to string

File: src/types/purchaseOrder.ts (hunk 7 above)
Comment (mllrdev-jerry-bayoneta) [reply src/types/purchaseOrder.ts]: done

Comment (mllrdev-jerry-bayoneta) [reply src/types/purchaseOrder.ts]: done

File: src/App.vue (hunk 1 above)
Comment (mllrdev-jerry-bayoneta) [reply src/App.vue]: removed text

File: src/shared/components/OrderItem.vue (hunk 6 above)
Comment (mllrdev-jerry-bayoneta) [reply src/shared/components/OrderItem.vue]: done

File: src/pages/Create.vue (hunk 5 above)
Comment (mllrdev-jerry-bayoneta) [reply src/pages/Create.vue]: done

File: src/pages/Create.vue (hunk 4 above)
Comment (mllrdev-jerry-bayoneta) [reply src/pages/Create.vue]: done

**Review by mllrdev-carlo-medel — APPROVED** [review]

**Pull request merged**